src/cljs/rems/spa.cljs

```diff
@@ -196,6 +196,7 @@
 ;;;; Pages
 (defn login-intro []
   [:div
+   [nav/logo]
```

Comment:
This is broken with LBR theme because the element is not full width.

Reply:
Let me check quickly

Reply:
yes, that is true, I am working on fixing it :)